plugin-script-deno/src/main/java/io/kestra/plugin/scripts/deno/Commands.java

```diff
@@ -0,0 +1,68 @@
+package io.kestra.plugin.scripts.deno;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.runners.TargetOS;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.scripts.exec.AbstractExecScript;
+import io.kestra.plugin.scripts.exec.scripts.models.ScriptOutput;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.util.List;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Execute Deno commands from the CLI."
+)
+@Plugin(examples = {
+    @Example(
+        full = true,
+        title = "Run a Deno script with network permissions.",
+        code = """
+            id: deno_permissions
+            namespace: company.team
+
+            tasks:
+              - id: deno_commands
+                type: io.kestra.plugin.scripts.deno.Commands
+                inputFiles:
+                  main.ts: |
+                    const response = await fetch("https://api.kestra.io/v1/ping");
+                    const data = await response.json();
+                    console.log(data);
+                commands:
```

Comment:
Maybe add a before command to install some dep to show it's possible in the example:

```suggestion
                beforeCommands:
                  - npm install something
```

What do you think @wrussell1999?